src/components/ViewContainer/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { getTokenCookie } from '../../utils/cookies';
+import jwt_decode from 'jwt-decode';
+
+const ViewContainer = () => {
+  const { role_id, ...rest } = jwt_decode(getTokenCookie());
```

Comment:
you can just deconstruct role_id, we're not using ...rest anywhere

Reply:
resolved in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/112/commits/3b533798007100cff693a4d3bd59d2c5f1588ba2